src/Symfony/Bridge/Doctrine/ArgumentResolver/EntityValueResolver.php

```diff
@@ -122,7 +128,7 @@ private function find(ObjectManager $manager, Request $request, MapEntity $optio
         }
     }
 
-    private function getIdentifier(Request $request, MapEntity $options, ArgumentMetadata $argument): mixed
+    private function getIdentifier(Request $request, MapEntity $options, ArgumentMetadata $argument, ParameterBag $valueBag): mixed
```

Comment:
As far as I can tell, the `$valueBag` parameter has effectively replaced the `$request` in most private methods you've touched here. Let's remove the now-unused parameter where we can.

```suggestion
    private function getIdentifier(ParameterBag $valueBag, MapEntity $options, ArgumentMetadata $argument): mixed
```